src/Fmt_ast.ml

```diff
@@ -1473,14 +1483,16 @@ and fmt_expression c ?(box= true) ?epi ?eol ?parens ?ext ({ast= exp} as xexp)
                is_simple c.conf (fun _ -> 0) (sub_exp ~ctx eI) ) ->
         let e1N = List.rev rev_e1N in
         let ctx'' = Exp eN in
-        hvbox 2
+        hvbox 4
```

Comment:
How would you feel about making this change only if at least one of the new options is enabled? The relative trade-off just does not seem like an overall win to me. It seems that there is much more code that is indented better with 2 than 4 than vice versa.

Reply:
seems reasonable